packages/build-info/src/workspaces/get-workspace-packages.ts

```diff
@@ -100,7 +100,8 @@ export async function getWorkspacePackages(project: Project, patterns: string[]
   const results = (
     await Promise.all(
       patterns.map((pattern) => {
-        const matcher = new Minimatch(pattern)
+        const cleanedPattern = pattern.replace(/^!?(\.\/)/, (match, p1) => match.replace(p1, ''))
```

Comment:
I wonder if anyone would ever want to use a path starting with `../`? 🤔

Reply:
As it's the root of the workspace I don't think it make any sense